features/step_definitions/style_guide_steps.rb

```diff
@@ -0,0 +1,8 @@
+When(/^I visit the style guide$/) do
+  visit '/style-guide'
+end
+
+Then(/^the page is rendered correctly$/) do
+  # check that header from layout has rendered
+  expect(page).to have_css('header.navbar')
```

Comment:
I think this is fine for now – we could perhaps abstract this later to have it look for a few elements on the page we deem enough to ensure the page is rendered.